pkg/sync/adapter.go

```diff
@@ -88,35 +100,25 @@ func (a Adapter) SyncFiles(syncParameters common.SyncParameters) (isPushRequired
 			forceWrite = true
 		}
 
-		// If the component already exists, sync only the files that changed
-		if syncParameters.ComponentExists {
-			// apply the glob rules from the .gitignore/.odo file
-			// and ignore the files on which the rules apply and filter them out
-			filesChangedFiltered, filesDeletedFiltered := util.FilterIgnores(ret.FilesChanged, ret.FilesDeleted, absIgnoreRules)
+		// apply the glob rules from the .gitignore/.odo file
```

Comment:
`.odoignore`

Reply:
we will resolve this in a follow up PR, for now I am merging this.

Reply:
addressed this in this PR https://github.com/openshift/odo/pull/3614
if there is nothing else wrong them can you please lgtm @mik-dass ?

Reply:
:+1: